packages/tasks/src/hardware.ts

```diff
@@ -72,6 +72,10 @@ export const SKUS = {
 				tflops: 12.0,
 				memory: [16],
 			},
+			"Quadro P6000": {
+				tflops: 20,
```

Comment:
```suggestion
				tflops: 12.63,
```

Using float32 numbers from https://www.techpowerup.com/gpu-specs/geforce-gtx-1070-ti.c3010